log/loggers/loggerInterface.go

```diff
@@ -1,39 +1,23 @@
 package loggers
 
-import "github.com/marcelocantos/frozen"
+import "github.com/arr-ai/frozen"
```

Comment:
(line below) `s/$/./` (ditto several other lines)

Any comment for an exported function will end up on godoc.org and should therefore be written as proper documentation, including correct grammar and punctuation.